src/backend/aspen/cli/db.py

```diff
@@ -229,101 +220,35 @@ def add_can_see(
 def create_phylo_run(
     ctx,
     group_name: str,
-    all_group_sequences: bool,
-    samples: Sequence[str],
-    aligned_gisaid_dump_id: Optional[int],
-    builds_template_file: str,
     builds_template_args: str,
-    tree_type: str,
     git_refspec: str,
+    tree_type: str,
 ):
     # these are injected into the IPython scope, but they appear to be unused.
     engine = ctx.obj["ENGINE"]
 
     with session_scope(engine) as session:
         group = session.query(Group).filter(Group.name == group_name).one()
 
-        all_samples: Iterable[Sample] = (
```

Comment:
This is legacy code that associated all county samples with each overview build. We don't need it anymore since we're *always* generating a fasta with all county samples.